etna/records/models.py

```diff
@@ -1,16 +1,16 @@
-from dataclasses import dataclass
+from dataclasses import dataclass, field
 
 from django.conf import settings
-from django.db import models
 from django.urls import reverse
 
-from wagtail.core.models import Page, Site
+from wagtail.core.models import Site
 
 from ..ciim.models import MediaManager, SearchManager
 from .transforms import transform_image_result, transform_record_page_result
 
 
-class RecordPage(Page):
+@dataclass
+class RecordPage:
```

Comment:
Maybe just `Record` would be a more appropriate name now?

Reply:
Agreed! That makes much more sense to me. I've pushed the change.